x-pack/docs/en/rest-api/security/update-api-key.asciidoc

```diff
@@ -1,5 +1,270 @@
 [role="xpack"]
 [[security-api-update-api-key]]
-=== Update API key information API
+=== Update API key API
 
-coming::[8.4.0]
+++++
+<titleabbrev>Update API key</titleabbrev>
+++++
+
+[[security-api-update-api-key-request]]
+==== {api-request-title}
+
+`PUT /_security/api_key/<id>`
+
+[[security-api-update-api-key-prereqs]]
+==== {api-prereq-title}
+
+* To use this API, you must have at least the `manage_own_api_key` cluster privilege.
+Users can only update their own API keys, i.e., API keys they created.
+To update another user's API key, you must use the run as feature.
+For more information, see <<run-as-privilege>>.
+
+IMPORTANT: It's not possible to use an API key as the authentication credential for this API.
+To update an API key, user password or <<security-api-get-token, token credentials>> are required.
+
+[[security-api-update-api-key-desc]]
+==== {api-description-title}
+
+The API keys created by <<security-api-create-api-key,create API Key>> or <<security-api-grant-api-key,grant API Key>> can be updated using this API.
+It's not possible to update expired API keys, or API keys that have been invalidated by <<security-api-invalidate-api-key,invalidate API Key>>.
+
+This API supports updates to an API key's access scope, i.e., its permissions, and its metadata.
```

Comment:
We should also explicitly list the auto-synced properties, especially the owner user's privileges. This makes the following callout more readable.

Reply:
I considered that as well, and am up for including them.

My reservation is that updating version and creator info is, in a sense, is an implementation detail -- these fields are not exposed via a public API and should be largely opaque to users.

Likewise, "updates to an API key's access scope" _includes_ the auto update to the user perm snapshot.

That said, I think including details about the auto-synced user perms makes a lot of sense. To avoid redundancy, I've tweak the text here as well as the callout. LMKWYT